lib/css-plugin/index.js

```diff
@@ -0,0 +1,61 @@
+const postcss = require('postcss');
+const concat = require('broccoli-concat');
+const Filter = require('broccoli-persistent-filter');
+
+CSStoJSON.prototype = Object.create(Filter.prototype);
+CSStoJSON.prototype.constructor = CSStoJSON;
+CSStoJSON.prototype.extensions = ['js'];
+CSStoJSON.prototype.targetExtension = 'js';
+
+function CSStoJSON(inputTree, options = {}) {
+  if (!(this instanceof CSStoJSON)) {
+    return new CSStoJSON(inputTree, options);
+  }
+
+  let mergedTree = concat(inputTree, {
+    outputFile: options.outputFile,
+    sourceMapConfig: { enabled: false },
+    allowNone: true
+  });
+
+  Filter.call(this, mergedTree, {
+    annotation: options.annotation
+  });
+}
+
+CSStoJSON.prototype.processString = function(content) {
+  return processConfigFile(content);
+};
+
+function processConfigFile(str) {
```

Comment:
The reason why I think it belongs in styles is because I think when we're adding our own app-specific styles, we might do something like this (which makes me want to put the "composes" definitions in the style sheets themselves):

```css
@composer {
  @my-component {
    composes: 'background-special';
  }
}

.background-special {
  background: red;
}
```